app/src/main/java/com/github/se/gomeet/viewmodel/UserViewModel.kt

```diff
@@ -153,4 +153,20 @@ class UserViewModel : ViewModel() {
       editUser(receiver!!.copy(followers = receiver.followers.minus(senderUid)))
     }
   }
+
+  /**
+   * Get the username of a user.
+   *
+   * @param uid The uid of the user.
+   * @return The username of the user.
+   */
+  suspend fun getUsername(uid: String): String? {
```

Comment:
This function will become simpler once we merge UserViewModel and AuthViewModel and add a private lateinit var currentUser:GoMeetUser to the class, but that's for later.